packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SignInMethodEditBox/index.module.scss

```diff
@@ -32,6 +32,15 @@
     padding: 0 _.unit(2);
     flex-grow: 1;
 
+    .checkBox {
+      flex-grow: 1;
+      width: 100%;
```

Comment:
nit: set display to block?

Reply:
No, we can't.
The `<CheckBox />` component's display prop is `flex`. Changing it to `block` will break the check box style.
<img width="245" alt="image" src="https://user-images.githubusercontent.com/10806653/200819891-d6dd0226-9448-4c87-a069-60212f44dff7.png">